docs/userguide/src/migration/prefix.md

```diff
@@ -75,6 +75,23 @@ See also:
 -   PR: <https://github.com/mmtk/mmtk-core/pull/1134>
 -   Example: <https://github.com/mmtk/mmtk-openjdk/pull/274>
 
+### Remove GC in `harness_begin`
```

Comment:
Because the versions are in the reversed order, we should list changes in the reversed order, too.